### ISSUE TITLE
Is the WebAuthenticatorBroker UI fixed?

### ISSUE THREAD

**Issue by ArchieCoder** [MicrosoftEdge/WebView2Feedback#988]
In WebView 1.0, the developer forgot to change the height of the WebAuthenticatorBroker when they moved from Windows 8 to Windows 10. It's very ugly.

I'm not sure if the WebAuthenticatorBroker is part of WebView2, but I'm taking a chance.

![image](https://user-images.githubusercontent.com/1608424/109204051-e1035080-7772-11eb-8a02-d0f154098647.png)

**Comment by champnic** [comment]
I'm not familiar with the WebAuthenticatorBroker, but looks like it would be a UWP control that uses WebView1 and would need to upgrade to use WebView2. I would maybe open an issue on the docs page for WebAuthenticatorBroker and ask the question there:
https://github.com/MicrosoftDocs/winrt-api/issues/new?title=&body=%0A%0A%5BEnter%20feedback%20here%5D%0A%0A%0A---%0A%23%23%23%23%20Document%20Details%0A%0A%E2%9A%A0%20*Do%20not%20edit%20this%20section.%20It%20is%20required%20for%20docs.microsoft.com%20%E2%9E%9F%20GitHub%20issue%20linking.*%0A%0A*%20ID%3A%203c758cb2-ad03-f28e-6947-21337462753a%0A*%20Version%20Independent%20ID%3A%203e818dc7-e032-c92b-80ba-0c0773391fec%0A*%20Content%3A%20%5BWebAuthenticationBroker%20Class%20(Windows.Security.Authentication.Web)%20-%20Windows%20UWP%20applications%5D(https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fuwp%2Fapi%2FWindows.Security.Authentication.Web.WebAuthenticationBroker%3Fview%3Dwinrt-19041)%0A*%20Content%20Source%3A%20%5Bwinrt-api-build%2Fxml%2FWindows.Security.Authentication.Web%2FWebAuthenticationBroker.xml%5D(https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fwinrt-api%2Fblob%2Fdocs%2Fwinrt-api-build%2Fxml%2FWindows.Security.Authentication.Web%2FWebAuthenticationBroker.xml)%0A*%20Product%3A%20**uwp**%0A*%20Technology%3A%20**winrt-reference**%0A*%20GitHub%20Login%3A%20%40lastnameholiu%0A*%20Microsoft%20Alias%3A%20**alholiu**

**Comment by ArchieCoder** [comment]
@champnic I did write it, but they bumped me to Q&A. I did write to Q&A, but they ignore it.

Thanks for taking the time to let us know about this. The scope of our feedback channel here on GitHub covers specific documentation fixes, but I can help redirect you to the right support channel to provide ypour feedback. The community of experts and Microsoft Engineers over at Microsoft Q&A offer timely answers to a multitude of technical questions surrounding windows-uwp. Hope your issue can get more visibility there.
